contracts/tansu/src/contract_tansu.rs

```diff
@@ -91,6 +97,8 @@ impl TansuTrait for Tansu {
     /// * `domain_contract` - The new domain contract
     fn set_domain_contract(env: Env, admin: Address, domain_contract: types::Contract) {
         auth_admin(&env, &admin);
+        // Enforce pause semantics for state-changing admin ops.
+        Self::require_not_paused(env.clone());
```

Comment:
No on the contrary, if there is an issue with these contracts we want to be able to do such admin ops in a paused state. The admin would pause the contract to prevent anything bad from happening then be able to adjust these.